homeassistant/components/lektrico/sensor.py

```diff
@@ -0,0 +1,322 @@
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        suggested_unit_of_measurement=UnitOfPower.KILO_WATT,
+        value_fn=lambda data: float(data["power_l3"]),
+    ),
+    LektricoSensorEntityDescription(
+        key="pf_l1",
+        translation_key="pf_l1",
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda data: float(data["power_factor_l1"]) * 100,
+    ),
+    LektricoSensorEntityDescription(
+        key="pf_l2",
+        translation_key="pf_l2",
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda data: float(data["power_factor_l2"]) * 100,
+    ),
+    LektricoSensorEntityDescription(
+        key="pf_l3",
+        translation_key="pf_l3",
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda data: float(data["power_factor_l3"]) * 100,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: LektricoConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Lektrico charger based on a config entry."""
+    coordinator = entry.runtime_data
+
+    sensors_to_be_used: tuple[LektricoSensorEntityDescription, ...]
+    if coordinator.device_type == Device.TYPE_1P7K:
+        sensors_to_be_used = SENSORS_FOR_CHARGERS + SENSORS_FOR_1_PHASE
+    elif coordinator.device_type == Device.TYPE_3P22K:
+        sensors_to_be_used = SENSORS_FOR_CHARGERS + SENSORS_FOR_3_PHASE
+    elif coordinator.device_type == Device.TYPE_EM:
+        sensors_to_be_used = (
+            SENSORS_FOR_LB_DEVICES + SENSORS_FOR_1_PHASE + SENSORS_FOR_LB_1_PHASE
+        )
+    elif coordinator.device_type == Device.TYPE_3EM:
+        sensors_to_be_used = (
+            SENSORS_FOR_LB_DEVICES + SENSORS_FOR_3_PHASE + SENSORS_FOR_LB_3_PHASE
+        )
+    else:
+        raise IntegrationError
+
+    async_add_entities(
+        LektricoSensor(
+            description,
+            coordinator,
+            f"{entry.data[CONF_TYPE]}_{entry.data[ATTR_SERIAL_NUMBER]}",
+        )
+        for description in sensors_to_be_used
+    )
+
+
+class LektricoSensor(LektricoEntity, SensorEntity):
+    """The entity class for Lektrico charging stations sensors."""
+
+    def __init__(
+        self,
+        description: LektricoSensorEntityDescription,
+        coordinator: LektricoDeviceDataUpdateCoordinator,
+        device_name: str,
+    ) -> None:
+        """Initialize Lektrico charger."""
+        super().__init__(coordinator, device_name)
+        self.entity_description: LektricoSensorEntityDescription = description
```

Comment:
Please overwrite the type outside of the constructor

Reply:
I made the change, thanks.